### ISSUE TITLE
Vignette code for pdp and ice method VI plots doesn't work

### ISSUE THREAD

**Issue by jtr13** [koalaverse/vip#135]
Thanks for a very useful package! 

FYI, these examples from the vignette don't work:

``` r
library(vip)
#> 
#> Attaching package: 'vip'
#> The following object is masked from 'package:utils':
#> 
#>     vi
set.seed(101)  # for reproducibility
trn <- as.data.frame(mlbench::mlbench.friedman1(500))  # ?mlbench.friedman1
pp <- ppr(y ~ ., data = trn, nterms = 11)  
vip(pp, method = "pdp") + ggtitle("PPR")
#> Error in match.arg(method): 'arg' should be one of "model", "firm", "permute", "shap"
vip(pp, method = "ice") + ggtitle("PPR")
#> Error in match.arg(method): 'arg' should be one of "model", "firm", "permute", "shap"
```

<sup>Created on 2023-04-14 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>

I can piece together from the help files what the code should be but it would be helpful to have the have the new syntax documented more clearly.

**Comment by bgreenwell** [comment]
Thanks for pointing this out @jtr13. I plan to dive back into this package in the next few weeks and the docs should be getting some much needed attention.

**Comment by jtr13** [comment]
Great to hear. If I can help let me know. We've been using **pdp**, **vip** and **fastshap** in the Interpretable Machine Learning class I'm teaching so triple thank you. I plan to write up some notes for my students on sticky points such as writing prediction functions so I'll share that in case it's useful for the docs.

**Comment by bgreenwell** [comment]
I'm glad to hear you find them useful! You can probably tell all three were written at different stages of my R career. @bradleyboehmke and I are starting to work on a [second edition of our book](https://bradleyboehmke.github.io/HOML/), so all three of those packages should be getting some much needed attention soon. Please let me know on any of the repos if you have suggestions on making them better! And yes, any thing you share will be more than useful.

**Comment by bgreenwell** [comment]
@jtr13 are the IML course materials by chance available online?

**Comment by jtr13** [comment]
Not yet... the plan is to organize them when the semester ends and then post publicly. That's a great offer... will take you up on that in the fall if you're available.

**Comment by bgreenwell** [comment]
Hi @jtr13, starting to fix up this package now that [fastshap](https://github.com/bgreenwell/fastshap is in good shape (and on to CRAN hopefully soon). In any case, I cannot find where in the docs or vignette those examples are? All the example in GH (and seemingly on CRAN) seem to be up to date and working.

**Comment by jtr13** [comment]
Good question.. I found it here: https://koalaverse.github.io/vip/articles/vip.html#pdp-method

**Comment by bgreenwell** [comment]
Ahh, thank you @jtr13. Long story short, when we updated the package, I had switched the URL in the repo to point to the R Journal article that we had just published which uses the correct code. But the old site still exists. In the next update the old pkg down site will be updated as well. Sorry for the confusion. Here’s the definitive guide for the current version: https://journal.r-project.org/archive/2020/RJ-2020-013/index.html
